AutoResizeFont/DynamicLabel.swift

```diff
@@ -10,38 +10,68 @@ import UIKit
 
 @IBDesignable
 final class DynamicLabel: UILabel {
+    
+    
+    
     @IBInspectable
-    var fontSizePts: CGFloat {
-        didSet {
+    var fontSize: CGFloat {
+        get{
+            return self.font!.pointSize
+        }
+        set{
+            self.font = UIFont(descriptor: .preferredFontDescriptor(withTextStyle: .body), size: newValue)
             calculateFontSize()
+        }
+    }
+    
+    
+    // Override font for view.
+    override var font : UIFont?{
+        
+        didSet{
+            setNeedsLayout()
             setNeedsDisplay()
+            self.sizeToFit()
         }
     }
     
+    // Deduced by multiplying the height of the Label by a constant.
+    var relativeHeight : CGFloat!
```

Comment:
Should be `relativeHeight: CGFloat`
- Don't think the `!` is necessary.
- Also, no spacing before the colon.

Reply:
the ! is neccesary or the compiler will give us errors in the init() call. 
spacing fixed in commit #b25317a

Reply:
I see. I understand the compiler error.